src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -753,7 +754,7 @@ public override async ValueTask<FormulaValue> Visit(RecordFieldAccessNode node,
 
         public override async ValueTask<FormulaValue> Visit(SingleColumnTableAccessNode node, EvalVisitorContext context)
         {
-            return CommonErrors.NotYetImplementedError(node.IRContext, "Single column table access");
+            throw new NotImplementedException();
```

Comment:
>     throw new NotImplementedException(); [](http://example.com/codeflow?start=7&length=41)

switching from a return error value  (soft error) to an exception (hard error) might be significant. Double checking if this is ok?